NEWS.md

```diff
@@ -1,5 +1,8 @@
 # dbplyr (development version)
 
+* `rows_patch()` no longer throws an error when more than one column needs to be
+  patched (@gorcha, #1443).
```

Comment:
This should mention that this fix is for `in_place = FALSE` and I think it is a bit nicer to state it positively ("now works" instead of "no longer throws an error")

```suggestion
* `rows_patch(in_place = FALSE)` now works when more than one column should be
  patched (@gorcha, #1443).
```